x-pack/packages/ml/date_picker/src/components/date_picker_wrapper.tsx

```diff
@@ -338,7 +338,7 @@ export const DatePickerWrapper: FC<DatePickerWrapperProps> = (props) => {
           <EuiToolTip content={tooltipMessage}>
             <EuiButton
               fill={false}
-              color={needsUpdate ? 'success' : 'primary'}
+              color={needsUpdate ? 'accentSecondary' : 'primary'}
```

Comment:
Is there another use of `success` that needs updating for the Update button here? I couldn't work out where the `success` class is coming from here:

<img width="1121" alt="Screenshot 2024-12-06 at 17 43 23" src="https://github.com/user-attachments/assets/722f89a7-b456-407d-816c-f17b9259a457">

Reply:
It looks like this is part of the `EuiSuperDatePicker` component we import from `@elastic/eui` so it's not code we are in control of, this needs to be updated within EUI.